homeassistant/components/dhcp/__init__.py

```diff
@@ -279,6 +279,9 @@ async def async_start(self):
         """Start watching for dhcp packets."""
         # Local import because importing from scapy has side effects such as opening
         # sockets
+        from scapy.arch import (  # pylint: disable=import-outside-toplevel  # noqa: F401
+            read_routes,
```

Comment:
I'd add an additional comment for this name since we don't use this name.

Reply:
Good idea 👍  Tweaked it a bit and added a comment.  Verified it still works